plugins/woocommerce-docs/src/Blocks/BlockConverter.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace WooCommerceDocs\Blocks;
+
+use League\CommonMark\Environment\Environment;
+use League\CommonMark\Extension\CommonMark\CommonMarkCoreExtension;
+use League\CommonMark\MarkdownConverter;
+
+/**
+ * Class MarkdownParser
+ */
+class BlockConverter {
+
+	/** // phpcs:ignore Generic.Commenting.DocComment.MissingShort
```

Comment:
Having some weird linter issue here where no matter how I format this comment it complains, so I have ignored it for now.